imagetest/test_suites/image_validation/cloud_test.go

```diff
@@ -0,0 +1,90 @@
+package imagevalidation
+
+import (
+	"io/ioutil"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const (
+	chronyService = "chronyd"
+	ntpService    = "ntp"
+	ntpdService   = "ntpd"
+)
+
+var ntpConfig = []string{"/etc/ntp.conf"}
+var chronyConfig = []string{"/etc/chrony.conf", "/etc/chrony/chrony.conf", "/etc/chrony.d/gce.conf"}
+
+// TestNTPService Verify that ntp package exist and configuration is correct.
+// debian 9, ubuntu 16.04 ntp
+// sles-12 ntpd
+// other distros chronyd
+func TestNTPService(t *testing.T) {
+	image, err := utils.GetMetadata("image")
+	if err != nil {
+		t.Fatalf("Couldn't get image from metadata")
+	}
+	var servicename string
+	var configPaths []string
+	switch {
+	case strings.Contains(image, "debian-9"):
```

Comment:
this is not the correct syntax for case statements in go. an image named "debian-9" would match this case, which has no code. then execution will move to the end of the switch block. so in essence, debian-9 and ubuntu-1604 would not get any servicename or configPaths variables set. did you run this code?

Reply:
Good Catch. This is totally different with Java